composables/useErc725.ts

```diff
@@ -39,17 +44,211 @@ const fetchProfile = async (profileAddress: Address): Promise<Profile> => {
     ? getImageUrlBySize(lsp3Profile.backgroundImage, 800, 400)
     : ''
 
+  const { getBalance } = useWeb3(PROVIDERS.RPC)
+  const balance = await getBalance(profileAddress)
+
   return {
     ...lsp3Profile,
     address: profileAddress,
     profileImageUrl: optimalProfileImage || '',
     backgroundImageUrl: optimalBackgroundImage || '',
+    balance,
+  }
+}
+
+const fetchAssets = async (profileAddress: Address, schema: string) => {
+  const erc725 = getInstance(
+    profileAddress,
+    LSP3UniversalProfileMetadata as ERC725JSONSchema[]
+  )
+  const result = await erc725.fetchData(schema)
```

Comment:
can this fail? does it need a try catch here? (maybe there is another parent one but im flaggin just in case)

Reply:
we use try/catch in upper level when calling `fetchAssets` function